include/mbedtls/config.h

```diff
@@ -148,6 +152,8 @@
  * \warning MBEDTLS_PLATFORM_XXX_ALT cannot be defined at the same time as
  * MBEDTLS_PLATFORM_XXX_MACRO!
  *
+ * \warning MBEDTLS_PLATFORM_TIME_ALT requires MBEDTLS_HAVE_TIME to be defined.
```

Comment:
We use `\warning` usually for security risks (line 152 is the exception), for prerequisites we use "Requires: ", like on line 126.